tests/playwright/src/specs/installation/update-install.spec.ts

```diff
@@ -42,6 +44,7 @@ test.afterAll(async ({ runner }) => {
 });
 
 test.describe.serial('Podman Desktop Update Update installation offering', () => {
+  test.skip(isLinux, 'Update is not supported on Linux');
```

Comment:
Move test.skip out of the describe, this way the check is validated before the beforeAll hook is run,